### ISSUE TITLE
Can't us fin RC candidate

### ISSUE THREAD

**Issue by croane** [docksal/docksal#731]
In `$HOME/.docksal/docksal.env` I added `DOCKSAL_USE_RC=1` and then ran `fin update`. However, fin doesn't appear to be updating. It still says it is on `fin version: 1.60.2` with fin version. Am I doing something wrong?

I want to test the platform cli integration to see if we want to use the RC version before this gets pushed to a stable version. Any help would be appreciated. Thanks!

**Comment by croane** [comment]
It seems the problem is with `fin update`. If I manually grab the release like this:

```
sudo curl -L https://raw.githubusercontent.com/docksal/docksal/v1.10.0-rc1/bin/fin -o /usr/local/bin/fin && \
sudo chmod +x /usr/local/bin/fin
```

`fin version` returns `fin version: 1.65.0`.... which looks correct. If I then run `fin update` again, it goes back to the old version. For the time being is this a suitable way to upgrade to the release candidate?

**Comment by achekulaev** [comment]
@croane yes there is a bug. We are going to fix it. Thanks for the repott.

**Comment by croane** [comment]
Thanks for your response. I was able to use the release candidate with the code above, so I think I am good for now. Good to know.

**Comment by croane** [comment]
Great news! I'm glad the release candidate got deployed. Thanks!
